### ISSUE TITLE
Multiple options for copying

### ISSUE THREAD

**Issue by mmahalwy** [quran/quran.com-frontend#1027]
**I'm submitting a ...**

- [ ] bug report
- [x] feature request
- [ ] other

**Current behavior:**
Copy for verse only copies the normal text and not other options.

**Expected / new behavior:**
Have other options for copying. Allowing copying only a few words

**Comment by ammarbinfaisal** [comment]
@mmahalwy What about double clicking a word(arabic text with class word) to copy it?

**Comment by mmahalwy** [comment]
@ammarbinfaisal I guess the intention here is people want to copy text, text with tashkeel, image, etc.

**Comment by mrtugicom** [comment]
It is a good habit to copy a whole 1 ayah, to avoid some misconception and fitnah when we cut the ayah.

But I would like to have option like check box whether i like to copi the arabic, or the translation, trasliteration, other translation, some of them or all of them.
With arabic at first, and other translation/literation on desired/current order.

**Comment by mrtugicom** [comment]
Yes, I think that's applicable too, seems good for mobile user to navigate
and for developers to implement, right?
